src/routes/faq/+page.svelte

```diff
@@ -1 +1,49 @@
-<div>FAQ_PAGE</div>
+<script lang="ts">
+  import MainnetVsTestnet from '$components/FaqAnswers/MainnetVsTestnet.svelte';
+  import TransactAsProvider from '$components/FaqAnswers/TransactAsProvider.svelte';
+  import type { ComponentType } from 'svelte';
+
+  let openItem: null | number = null;
+
+  const questions: { q: string; a: ComponentType }[] = [
+    {
+      q: 'What is the difference between Mainnet and Testnet (Rococo)?',
+      a: MainnetVsTestnet,
+    },
+    {
+      q: 'How do I transact on Frequency as a provider?',
+      a: TransactAsProvider,
+    },
+  ];
+</script>
+
+<div class="content-block w-single-block text-left text-base">
+  <h2 class="section-title">FAQ's</h2>
+  {#each questions as item, index}
```

Comment:
To clarify my comment in the previous review, I think this should be it's own component:

e.g.  something similar to this:
 ```
<FAQ>
   <FAQEntry q="What is this?" a="It is this." />
   <FAQEntry q="What is that?" a="That's another question." />
 </FAQ>
```